### PR TITLE
Update change log

### PR DESCRIPTION
**Pull request by ogaca-dd**
### What does this PR do?

Update CHANGELOG for 7.16.0.
I also replaced ` by ``.

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,100 @@
...
+======
+
+.. _Release Notes_7.16.0_Prelude:
+
+Prelude
+-------
+
+This release introduces major version 7 of the Datadog Agent, which starts at v7.16.0. The only change from Agent v6 is that
+v7 defaults to Python 3 and only includes support for Python 3. Before upgrading to v7, confirm that any
+custom checks you have are compatible with Python 3. See this ``guide <https://docs.datadoghq.com/agent/guide/python-3/>``_
```
Comment (olivielpeau) [inline CHANGELOG.rst]: I don't understand why there are double backticks here, there are single backticks on the original release note file: https://github.com/DataDog/datadog-agent/blob/master/releasenotes/notes/agent-7-prelude-73db5a64808ff94b.yaml

Anyway, double backticks make this non-valid reST, so let's update this to:
```suggestion
custom checks you have are compatible with Python 3. See this `guide <https://docs.datadoghq.com/agent/guide/python-3/>`_
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,100 @@
...
+- Add ``ignore_autodiscovery_tags`` parameter config check.
+  
+  In some cases, a check should not receive tags coming from the autodiscovery listeners.
+  By default ``ignore_autodiscovery_tags`` is set to false which doesn't change the behavior of the checks.
+  The first check that will use it is ``kubernetes_state``.
+
+- Adds a new ``flare_stripped_keys`` config setting to clean up additional
+  configuration information from flare.
+
+- Adding a new config option "exclude_gce_tags", to configure which metadata
```
Comment (olivielpeau) [inline CHANGELOG.rst]: nit: 
```suggestion
- Adding a new config option ``exclude_gce_tags``, to configure which metadata
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -2,6 +2,100 @@
...
+
+- Check that cluster-name provided by configuraiton file are compliant with the same rule as on GKE. Logs an error and ignore it otherwise.
+
+
+.. _Release Notes_7.16.0_Other Notes:
+
+Other Notes
+-----------
+
+- JMXFetch upgraded to ``0.33.1 <https://github.com/DataDog/jmxfetch/releases/0.33.1>``_
```
Comment (olivielpeau) [inline CHANGELOG.rst]: ditto:

```suggestion
- JMXFetch upgraded to `0.33.1 <https://github.com/DataDog/jmxfetch/releases/0.33.1>`_
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -2,6 +2,100 @@
...
+-------
+
+This release introduces major version 7 of the Datadog Agent, which starts at v7.16.0. The only change from Agent v6 is that
+v7 defaults to Python 3 and only includes support for Python 3. Before upgrading to v7, confirm that any
+custom checks you have are compatible with Python 3. See this ``guide <https://docs.datadoghq.com/agent/guide/python-3/>``_
+for more information.
+
+Except for the supported Python versions, v7.16.0 and v6.16.0 have the same features.
+
+
```
Comment (olivielpeau) [inline CHANGELOG.rst]: actually, we seem to be missing the prelude that links to the tag on the `integrations-core` repo and mentions the release date?

Whether it was added or not in a separate release note file (as it's supposed to be), feel free to just add the relevant content directly in the changelog here. No need to add the file in `releasenotes/` now if it's not there

**Review by olivielpeau — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst [hunk 5]
```diff
@@ -2,6 +2,100 @@
...
+- APM: Added a fallback into the SQL obfuscator to handle SQL engines that treat
+  backslashes literally.
+
+- The default list of sensitive keywords for process argument scrubbing now uses wildcards before and after.
+
+- On Windows process agent, fix problem wherein if the agent is unable
+  to figure out the process' user name, the process info/stats were not
+  sent at all.  Now sends all relevant stats without the username
+
+- On windows, correctly deletes python 3 precompiled files (py3) in
```
Comment (KSerrania) [inline CHANGELOG.rst]: I think the author of this release note (cc @derekwbrown)  meant:
```suggestion
- On windows, correctly deletes python 3 precompiled files (.pyc) in
```
?
But it's not a big deal, we can merge as-is.

File: CHANGELOG.rst [hunk 6]
```diff
@@ -2,6 +2,104 @@
...
+Release on: 2019-12-18
+
+This release introduces major version 7 of the Datadog Agent, which starts at v7.16.0. The only change from Agent v6 is that
+v7 defaults to Python 3 and only includes support for Python 3. Before upgrading to v7, confirm that any
+custom checks you have are compatible with Python 3. See this `guide <https://docs.datadoghq.com/agent/guide/python-3/>`_
+for more information.
+
+Except for the supported Python versions, v7.16.0 and v6.16.0 have the same features.
+
+- Please refer to the `7.16.0 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7160>`_ for the list of changes on the Core Checks
```
Comment (olivielpeau) [inline CHANGELOG.rst]: nit: no need to use a list item here I think, it looks a bit weird 😅 

```suggestion
Please refer to the `7.16.0 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-7160>`_ for the list of changes on the Core Checks
```

**Review by olivielpeau — DISMISSED** [review]
LGTM, just one nit

**Review by olivielpeau — APPROVED** [review]

**Pull request merged**